snakemake_executor_plugin_googlebatch/executor.py

```
@@ -519,9 +523,61 @@ async def check_active_jobs(self, active_jobs: List[SubmittedJobInfo]):
+    ):
+        """
+        Download logs using Google Cloud Logging API and save
+        them locally. Since tail logging does not work, this function
+        is run only at the end of the job.
+        """
+        job_uid = job_info.aux["batch_job"].uid
+        filter_query = f"labels.job_uid={job_uid}"
+        logfname = job_info.aux["logfile"]
+
+        log_client = logging.Client(project=self.executor_settings.project)
+        logger = log_client.logger("batch_task_logs")
```

Ruff (0.8.2)

Local variable `logger` is assigned to but never used

Remove assignment to unused variable `logger`

(F841)